modules/spo-container-profile-instances.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-container-profile-instances_{context}"]
+= Merging per-container profile instances
+
+By default, each container instance records into a separate profile. The Security Profiles Operator can merge the per-container profiles into a single profile.
```

Comment:
We could amend this with another sentence that describes when this is important.

```
The Security Profiles Operator can merge the per-container profiles into a single profile.
This is useful when deploying applications using `ReplicaSet` or `Deployments`.
```